src/ruby/spec/channel_credentials_spec.rb

```diff
@@ -60,6 +60,31 @@ def load_test_certs
       blk = proc { GRPC::Core::ChannelCredentials.new(nil, '', nil) }
       expect(&blk).to raise_error
     end
+
+    it 'can be constructed with a fallback credential' do
```

Comment:
How about another test for insecure channel credentials?

Reply:
Ruby/ext doesn't have insecure channel credentials. Do you mean insecure channel + call credential, like the php case we're talking about this week? if so, I think call credentials should have covered that kind of case already. I'll check.

Reply:
Never mind. I forgot about Ruby not having insecure channel creds.

Reply:
got it